src/services/MoxfieldService.ts

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import { useCallback } from "react";
+
+const useGetMoxfieldProfile = () => {
+    const endpoint = "https://chatterfang.onrender.com/moxfield/";
+
+    return useCallback(async (moxfieldId: string) => {
+        try {
+            const _res = await axios.get<MoxfieldResponse>(endpoint + moxfieldId, {
+                headers: { "Content-Type": "application/json" }
+            });
+            return _res.data;
```

Comment:
I recommend that we do not return the service type out of the box. instead, create a new "MoxfieldProfile" domain/type, and then convert the MoxfieldResponse type to that. Even if it is almost 1:1, this prevents us from having external contracts bleed into our UI code.